src/resources/parser/glb-parser.js

```diff
@@ -919,9 +919,12 @@ var createMaterial = function (gltfMaterial, textures, disableFlipV) {
 
     var material = new StandardMaterial();
 
-    // glTF dooesn't define how to occlude specular
+    // glTF doesn't define how to occlude specular
     material.occludeSpecular = true;
 
+    // glTF doesn't expect material opacity to fade specular lighting
+    material.opacityFadesSpecRefl = true;// false
```

Comment:
Delete `// false`

Reply:
I will remove these changes from this PR - and add to a subsequent PR to discuss whether PBR glTF materials should fade specular or not.

Reply:
removed changes